C++/13-Bit Manipulation/6-is_power_of_2.cpp

```diff
@@ -0,0 +1,21 @@
+		#include<bits/stdc++.h>
```

Comment:
Is this related to `Bits` or `DSA`? I guess this one should be in the programming fundamentals. What do you think?

Reply:
Hey @ShehrozIrfan 

No. Why are you bothering for small things. This is basically an advance header file. If you include this header file then all the header files will be imported automatically.

Reply:
Hey @tarunsamanta2k20 

My apologies for not making the point clear.

Actually, I was not asking about the header file, I was asking about the file like it says the `program to check if the integer is the power of 2`, so I was wondering if we can add this to `programming fundamentals` or not.
What do you think?

Reply:
@ShehrozIrfan 

No need to add to programming fundamental. This bit manipulation topic is under DSA then you must add to your DSA branch i.e learn-dsa. Merge it now. I will share more DSA topic in future.